features/features.go

```diff
@@ -57,6 +57,13 @@ var (
 				enableIn(configv1.DevPreviewNoUpgrade).
 				mustRegister()
 
+	FeatureGateSetEIPForNLBIngressController = newFeatureGate("SetEIPForNLBIngressController").
+							reportProblemsToJiraComponent("Routing").
```

Comment:
Should this be "Networking / router" to match the Jira component, or does openshift/api do some internal mapping (which isn't mentioned in the godoc for `featureGateBuilder`)?  This is probably a @JoelSpeed question.

Reply:
Fair enough.  "Routing" is neither a Jira project nor an OCPBUGS component name.  https://github.com/openshift/api/compare/f190c684d393f69233f4eed3f8b59ecb5db116fb..2e1e160f29698d1b8635ba6db37fa0e591c58850 changed the feature gate to reference "Networking / router" instead.  Thanks!